contributing/development/core_and_modules/scripting_development.rst

```diff
@@ -0,0 +1,31 @@
+.. _doc_scripting_development:
+
+Scripting development
+=====================
+
+GDScript
+--------
+
+Annotation guidelines
+~~~~~~~~~~~~~~~~~~~~~
+
+..
+    This description intentionally avoids mention of implementation and
+    compilation details because these are often inconsistent between annotations
+
+Create annotations for modifiers that act on the script or its code.
+Addionally, create annotations for behaviour that is specific to the Godot
```

Comment:
```suggestion
Addionally, create annotations for behavior that is specific to the Godot
```

Reply:
I unmarked this as resolved since the PR still have the previous word.